pkg/selector/outputs/sortingView.go

```diff
@@ -0,0 +1,258 @@
+func createListItems() *[]list.Item {
+	shorthandFlags := []string{
+		sorter.GPUCountField,
+		sorter.InferenceAcceleratorsField,
+		sorter.VCPUs,
+		sorter.Memory,
+		sorter.GPUMemoryTotal,
+		sorter.NetworkInterfaces,
+		sorter.SpotPrice,
+		sorter.ODPrice,
+		sorter.InstanceStorage,
+		sorter.EBSOptimizedBaselineBandwidth,
+		sorter.EBSOptimizedBaselineThroughput,
+		sorter.EBSOptimizedBaselineIOPS,
+	}
+
+	items := []list.Item{}
+
+	for _, flag := range shorthandFlags {
+		items = append(items, item(flag))
+	}
+
+	return &items
+}
+
+// resizeSortingView will change the dimensions of the sorting view
+// in order to accommodate the new window dimensions represented by
+// the given tea.WindowSizeMsg
+func (m sortingModel) resizeView(msg tea.WindowSizeMsg) sortingModel {
+	shorthandList := &m.shorthandList
+	shorthandList.SetWidth(msg.Width)
+	// ensure that text input is right below last option
+	if msg.Height >= len(shorthandList.Items())+sortingTitlePadding+sortingFooterPadding {
+		shorthandList.SetHeight(len(shorthandList.Items()) + sortingTitlePadding)
+	} else if msg.Height-sortingFooterPadding-sortDirectionPadding > 0 {
+		shorthandList.SetHeight(msg.Height - sortingFooterPadding - sortDirectionPadding)
+	} else {
+		shorthandList.SetHeight(1)
+	}
+
+	// ensure cursor of list is still hidden after resize
+	if m.sortTextInput.Focused() {
+		shorthandList.Select(len(m.shorthandList.Items()))
+	}
+
+	m.shorthandList = *shorthandList
+
+	return m
+}
+
+// update updates the state of the sortingModel
+func (m sortingModel) update(msg tea.Msg) (sortingModel, tea.Cmd) {
+	var cmd tea.Cmd
+	var cmds []tea.Cmd
+
+	switch msg := msg.(type) {
+	case tea.KeyMsg:
+		switch msg.String() {
+		case "down":
+			if m.shorthandList.Index() == len(m.shorthandList.Items())-1 {
+				// focus text input and hide cursor in shorthand list
+				m.shorthandList.Select(len(m.shorthandList.Items()))
+				m.sortTextInput.Focus()
+			}
+		case "up":
+			if m.sortTextInput.Focused() {
+				// go back to list from text input
+				m.shorthandList.Select(len(m.shorthandList.Items()))
+				m.sortTextInput.Blur()
+			}
+		case "tab":
+			m.isDescending = !m.isDescending
+		}
+
+		if m.sortTextInput.Focused() {
+			m.sortTextInput, cmd = m.sortTextInput.Update(msg)
+			cmds = append(cmds, cmd)
+		}
+	}
+
+	if !m.sortTextInput.Focused() {
+		m.shorthandList, cmd = m.shorthandList.Update(msg)
+		cmds = append(cmds, cmd)
+	}
+
+	return m, tea.Batch(cmds...)
+}
+
+// view returns a string representing the sorting view
+func (m sortingModel) view() string {
+	outputStr := strings.Builder{}
+
+	// draw sort direction
+	outputStr.WriteString(sortDirectionStyle.Render("Sort Direction:"))
+	outputStr.WriteString(" ")
+	if m.isDescending {
+		outputStr.WriteString(descendingStyle.Render(descendingText))
+	} else {
+		outputStr.WriteString(ascendingStyle.Render(ascendingText))
+	}
+	outputStr.WriteString("\n")
+	outputStr.WriteString("\n")
```

Comment:
You could just write `outputStr.WriteString("\n\n")` :P

Reply:
Good point. I fixed it in [4db1c19](https://github.com/aws/amazon-ec2-instance-selector/pull/152/commits/4db1c194f8616ab3c6bf6d2ba53ddf44097f5f22)